modules/op-running-pipeline-run-using-pipelines-as-code.adoc

```diff
@@ -0,0 +1,69 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: REFERENCE
+[id="running-pipeline-run-using-pipelines-as-code_{context}"]
+= Running a pipeline run using {pac} 
+
+[role="_abstract"]
+With default configuration, {pac} runs any pipeline run in the `.tekton/` directory of the default branch of repository, when specified events such as pull request or push occurs on the repository. For example, if a pipeline run on the default branch has the annotation `pipelinesascode.tekton.dev/on-event: "[pull_request]"`, it will run whenever a pull request event occurs.
+
+In the event of a pull request or a merge request, {pac} also runs pipelines from branches other than the default branch, if the following conditions are met by the author of the pull request:
+
+* The author is the owner of the repository.
+* The author is a collaborator on the repository.
+* The author is a public member on the organization of the repository.
+* The pull request author is inside an `OWNER` file located in the repository root on the `main` branch (as defined in the GitHub configuration for the repository), and added to either `approvers` or `reviewers` sections. For example, if the author `chmouel` is listed in the `approvers` section, then a pull request raised by `chmouel` starts the pipeline run.
+
+[source,yaml]
```

Comment:
We can make this code part of the previous bullet.

Reply:
you can change chmouel to something more generic as name like john or whatever :)

Reply:
I was also thinking to change the name. Please make the approver's name generic.

Reply:
Updated the name to `john`
Thanks!